### PR TITLE
Implement core apis

### PR DESCRIPTION
**Pull request by ashwin1111**

### CODE REVIEW

File: apps/partner/views.py [hunk 1]
```diff
@@ -1,3 +1,61 @@
...
+
+from .helpers import get_cluster_domain
+from .serializers import PartnerOrgSerializer
+
+
+User = get_user_model()
+
+
+class PartnerView(generics.GenericAPIView):
+    def post(self, request):
```
Comment (ashwin1111) [inline apps/partner/views.py:19]: move all logic to serializer

File: apps/partner/helpers.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import json
+from typing import Dict
+
+import requests
+
+from django.conf import settings
+
+
+def post_request(url: str, body: Dict, access_token: str):
```
Comment (Shwetabhk) [inline apps/partner/helpers.py:9]: returns Dict

File: apps/partner/views.py [hunk 3]
```diff
@@ -1,3 +1,61 @@
...
+from apps.partner.models import PartnerOrg
+
+from .helpers import get_cluster_domain
+from .serializers import PartnerOrgSerializer
+
+
+User = get_user_model()
+
+
+class PartnerView(generics.GenericAPIView):
```
Comment (Shwetabhk) [inline apps/partner/views.py:18]: should be called Partner Org View

File: apps/partner/urls.py [hunk 4]
```diff
@@ -1,3 +1,7 @@
 from django.urls import path
 
-urlpatterns = []
+from .views import PartnerView
+
+urlpatterns = [
+    path('', PartnerView.as_view(), name='partner')
```
Comment (Shwetabhk) [inline apps/partner/urls.py:6]:
```suggestion
    path('', PartnerView.as_view(), name='partner_orgs')
```

File: apps/partner/urls.py [hunk 5]
```diff
@@ -1,3 +1,7 @@
 from django.urls import path
 
-urlpatterns = []
+from .views import PartnerView
```
Comment (Shwetabhk) [inline apps/partner/urls.py:3]:
```suggestion
from .views import PartnerOrgView
```

File: apps/partner/views.py (hunk 3 above)
Comment (Shwetabhk) [inline apps/partner/views.py:18]:
```suggestion
class PartnerOrgView(generics.RetrieveUpdateAPIView):
```

File: apps/users/helpers.py [hunk 6]
```diff
@@ -0,0 +1,19 @@
+from django.conf import settings
```
Comment (Shwetabhk) [inline apps/users/helpers.py:1]: Create a `fyle` django app and move there

File: apps/users/views.py [hunk 7]
```diff
@@ -1,3 +1,27 @@
...
+from rest_framework.response import Response
 
-# Create your views here.
+from fyle_rest_auth.models import AuthToken
+
+from apps.partner.models import PartnerOrg
+from .helpers import PlatformConnector
+
+
+class UserProfileView(generics.GenericAPIView):
```
Comment (Shwetabhk) [inline apps/users/views.py:11]: I think we also have a RetrieveAPIView

Comment (Shwetabhk) [reply apps/users/views.py:11]: lets use that

**Review by Shwetabhk — COMMENTED** [review]
Please checkout the comments and make changes

File: apps/users/helpers.py [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+
+from fyle.platform import Platform
+
+class PlatformConnector:
```
Comment (ashwin1111) [inline apps/users/helpers.py:5]: move to fyle app

**Review by Shwetabhk — APPROVED** [review]

**Pull request merged**